src/resources/postcss/views/_list.pcss

```diff
@@ -50,27 +66,37 @@
 	.tribe-events-calendar-list__event--description {
 		@mixin desktop-body-1;
 		margin-bottom: var(--spacer-4);
+		@media (--viewport-medium) {
+			@mixin desktop-body-1;
+		}
 	}
 
 	.tribe-events-calendar-list__event--featured-image {
-		flex: 1 0 320px;
+		flex: 1 1 100%;
+		margin-bottom: var(--spacer-4);
 		@media (--viewport-medium) {
 			flex: 1 0 320px;
 			margin-left: var(--spacer-6);
+			margin-bottom: 0;
+		}
+
+		img {
```

Comment:
I used it this way because if they override the template without adding the class to the image, their image won't be mobile friendly. I would prefer to keep it this way. This could save some support tickets and it shouldn't be causing any harm

Reply:
In this regard I have to disagree. Classes for everything that gets styles has a couple of advantages:

1) Consistency, the end themer knows that anything getting tribe styles uses a class, with consistent and semantic naming, always.
2) Structure becomes completely malleable for the end themer who is customizing, as does element types.

Here we are locking them into a structure and locking the element type.

Reply:
I agree with Sam and Paul here.